wpimath/src/test/python/kinematics/test_swerve_module_state.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+import math
+
+from wpimath.kinematics import SwerveModuleState
+from wpimath.geometry import Rotation2d
+
+
+kEpsilon = 1e-9
+
+
+def test_optimize():
+    angle_a = Rotation2d.fromDegrees(45)
+    ref_a = SwerveModuleState(speed=-2, angle=Rotation2d.fromDegrees(180))
```

Comment:
nit: given the components of these vectors have different types, I think it's clear enough from the types alone what each param is